src/backend/src/health/health.controller.ts

```diff
@@ -0,0 +1,19 @@
+import { Controller, Get, HttpStatus } from '@nestjs/common';
+import {
+  HealthCheckService,
+  HttpHealthIndicator,
+  HealthCheck,
+} from '@nestjs/terminus';
+import { ApiExcludeEndpoint } from '@nestjs/swagger';
+@Controller('health')
+export class HealthController {
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  constructor() {}
+
+  @Get()
+  @HealthCheck()
+  @ApiExcludeEndpoint()
+  async check() {
```

Comment:
Can you please add Swagger Annotations and return type here?

Reply:
I thought may be this health check do not need to expose, but sure, I can remove  ApiExcludeEndpoint() and add it to swagger